test/quantization/test_quantize_jit.py

```diff
@@ -2272,6 +2272,10 @@ def __init__(self):
 
             def forward(self, x):
                 x = self.conv(x)
+                # add_scalar
```

Comment:
I thought this was already supported. What was missing?

Reply:
the fusion pattern is inconsistent